tests/reference_emails/submitted.html

```diff
@@ -50,6 +50,11 @@
         &#34;Iiwicm9sZXMiOiJnZW5lcmFsIiwiZXhwIjoxNjI5MzkxNjY1LCJ0ZW0iOjAsIm1zdG91dCI6dHJ1ZSwibXNzdWIiO&#34;<br>
         &#34;nRydWUsImludHN1YiI6NX0.GS7hvnJCPa3i1e1g9JwLsZ-ol-4frLhfvJC0WRBxqH4&#34;<br>
     ),<br>
+    &#34;src_name&#34;: &#34;test&#34;,<br>
+    &#34;RA&#34;: 0.0,<br>
+    &#34;DEC&#34;: 0.0,<br>
+    &#34;T1&#34;: &#34;2001-12-11T00:00:00.000&#34;,<br>
```

Comment:
why these are not the defaults?

Reply:
This template, I think, could be removed, since is no longer needed.